.github/workflows/build.yml

```diff
@@ -19,17 +20,147 @@ jobs:
           toolchain: nightly-2021-05-11
           override: true
           target: wasm32-unknown-unknown
-    - name: Install llvm-10
-      run: sudo apt-get install llvm-10 clang-10
+    - name: Install System Level Dependencies
+      run: sudo apt-get install build-essential ocaml ocamlbuild automake autoconf libtool wget python libssl-dev git cmake perl pkg-config curl llvm-10 clang-10 libclang-10-dev
     - uses: actions/checkout@v2
       with:
         submodules: 'true'
-    - name: Build
+        
+    - name: Build core blockchain
       run: cargo build --verbose --release
-    - name: Save binaries
-      uses: actions/upload-artifact@v1
+    - name: Save core-blockchain binaries
+      uses: actions/upload-artifact@v2
       with:
-        name: binaries
-        path: target/release/phala-node
-    - name: Tests
+        name: core-blockchain-binaries
+        path: ./target/release/phala-node
+    - name: Save pherry binaries
+      uses: actions/upload-artifact@v2
+      with:
+        name: pherry-binaries
+        path: ./target/release/pherry
```

Comment:
Can we save the two binaries in a single step?

Reply:
Done